src/services/wallets/metamask/index.ts

```diff
@@ -2,13 +2,18 @@ import * as metamask from "./metamask";
 import { Wallet } from "../wallet";
 
 const metamaskWallet: Wallet = {
+  icon:
+    "https://cdn.iconscout.com/icon/free/png-512/metamask-2728406-2261817.png",
   login: async () => {
     if (!metamask.isInstalled()) throw new Error("Metamask not installed");
     return metamask.getWalletAddress();
   },
   logout: () => {
     return;
   },
+  reload: () => {
```

Comment:
It actually wasn't left out. The current way I'm implementing the Wallet API means I'm creating an interface that will have far more actions it will need to handle than any one wallet will likely implement. 

In this case wallets like Torus which are embedded in the site itself need to be able to reload while Wallets in the browser won't. As a result the function for implementing the reload for Metamask is empty.